test/inductor/test_torchinductor.py

```diff
@@ -10358,6 +10358,32 @@ def fn(x):
         x = torch.rand([4, 4, 3], dtype=torch.float64)
         self.common(fn, (x,))
 
+    @skipCUDAIf(not SM80OrLater, "uses bfloat16 which requires SM >= 80")
+    def test_dtypeview(self):
+        # https://github.com/pytorch/pytorch/issues/126338
+        @torch.compile
+        def fn(x, y):
+            x = x.view(torch.float16)
+            y = y.view(torch.float16) + 1
```

Comment:
add one where there is a use before, not just after

Reply:
added in https://github.com/pytorch/pytorch/pull/128883/commits/c638822f72e70db19ae20b044f09ccb6941255a5